core/arch/arm/kernel/boot.c

```diff
@@ -1252,6 +1252,10 @@ void init_tee_runtime(void)
 	if (!IS_ENABLED(CFG_VIRTUALIZATION))
 		call_preinitcalls();
 	call_initcalls();
+
+	/* crypto_rng_read() should be ready to be used by now */
```

Comment:
not sure this comment adds a lot of value.

Reply:
It answers the question of why this function is called so late, one would normally expect this to be called earlier.

Reply:
Ah now it makes sense. Can we mention this explicitly perhaps? I don't think it is obvious